plugins/jobs/web_client/routes.js

```diff
@@ -6,25 +6,26 @@ import JobDetailsWidget from './views/JobDetailsWidget';
 import JobListWidget from './views/JobListWidget';
 
 router.route('job/:id', 'jobView', function (id) {
-    var job = new JobModel({_id: id}).once('g:fetched', function () {
+    var job = new JobModel({ _id: id }).once('g:fetched', function () {
         events.trigger('g:navigateTo', JobDetailsWidget, {
             job: job,
             renderImmediate: true
         });
     }, this).once('g:error', function () {
-        router.navigate('collections', {trigger: true});
+        router.navigate('collections', { trigger: true });
     }, this);
     job.fetch();
 });
 
 router.route('jobs/user/:id', 'jobList', function (id) {
     events.trigger('g:navigateTo', JobListWidget, {
-        filter: {userId: id}
+        filter: { userId: id }
     });
 });
 
-router.route('jobs', 'allJobList', function () {
+router.route('jobs(/:view)', 'allJobList', function (view) {
```

Comment:
@zachmullen, Thank you for reviewing this PR. 
This route doesn't break the old behavior because the view part is inside a pair of parentheses. Actually, the default landing URL to the page from other pages remained the same.